.env.example

```diff
@@ -35,9 +35,19 @@ MICROSOFT_SCOPE="user.read"
 MICROSOFT_TENANT="common"
 
 # Mailer config
-MAILER_SMTP_URL="smtps://user@domain.com:pass@smtp.domain.com"
+MAILER_SMTP_ENABLE="true"
+MAILER_USE_ADVANCE_CONFIGS="false"
 MAILER_ADDRESS_FROM='"From Name Here" <from@example.com>'
 
+MAILER_SMTP_URL="smtps://user@domain.com:pass@smtp.domain.com" # if MAILER_USE_ADVANCE_CONFIGS is true
+
+MAILER_SMTP_HOST="smtp.domain.com"      # if MAILER_USE_ADVANCE_CONFIGS is false
+MAILER_SMTP_PORT="587"                  # if MAILER_USE_ADVANCE_CONFIGS is false
+MAILER_SMTP_SECURE="true"               # if MAILER_USE_ADVANCE_CONFIGS is false
+MAILER_SMTP_USER="user@domain.com"      # if MAILER_USE_ADVANCE_CONFIGS is false
+MAILER_SMTP_PASSWORD="pass"             # if MAILER_USE_ADVANCE_CONFIGS is false
+MAILER_TLS_REJECT_UNAUTHORIZED="true"   # if MAILER_USE_ADVANCE_CONFIGS is false
```

Comment:
Do all these have to be strings, can some of them normal booleans and numbers and we do a `toString()` conversion in the infra-config module anyway??

Reply:
The default behavior is process.env always reads the env variable as string regarding of string/boolean.
No matter if you add an inverted comma or not, process.env reads it as a string.